modules/controller/lib/source/read.js

```diff
@@ -1,5 +1,21 @@
+const throwIfNoModel = require('../throw_if_no_model');
 const getParams = require('../get_params');
 
 module.exports = function(source, opts, request) {
-  return source.read(getParams(request, opts));
+  var relation = request.params.relation || null;
+  var params = getParams(request, opts);
+  var filterIds = request.query.filter && request.query.filter.id;
+
+  if (relation) {
+    var findRelatedModels = source.related.bind(source, relation);
+    return source.byId(request.params.id, relation).
+      then(throwIfNoModel).
+      then(findRelatedModels).
```

Comment:
Seems the error's in here somewhere. Breaking this out into an anonymous, I can log inside it, but never get anything loggin in `spread`